services/QuillLMS/client/app/bundles/Teacher/components/admin_access/adminTable.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react'
```

Comment:
Just wanted to say that I like that you've broken this out into a stand-alone component.